libs/trainer/bbhnet/trainer/trainer.py

```diff
@@ -180,10 +180,17 @@ def train(
     # TODO: Allow different loss functions or optimizers to be passed?
     criterion = torch.nn.functional.binary_cross_entropy_with_logits
     optimizer = torch.optim.Adam(
-        model.parameters(), lr=lr, weight_decay=weight_decay
+        model.parameters(), lr=max_lr, weight_decay=weight_decay
```

Comment:
I presume the `lr` set here is discarded, (and not used as the starting `lr`), but wanted to clarify.

Reply:
Yeah my understanding is the scheduler just disregards whatever the optimizers existing LR is and sets it based on its schedule (in this case it gets initialized to `max_lr / div_factor`, where `div_factor` defaults to 25)